aicsimageio/readers/bfio_reader.py

```diff
@@ -181,8 +181,7 @@ def set_scene(self, scene_id: Union[str, int]) -> None:
                     "Scene id: Cannot change scene for "
                     + f"{self.__class__.__name__} objects."
                 )
-
-        else:
+        elif scene_id is not None:
```

Comment:
is this fixing a real bug in the bfio reader?

Reply:
Not a bug-just changes behavior when passing a null scene id